public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -964,12 +964,43 @@ export class ElasticDatasource
         if (expression.length > 0) {
           expression += ' AND ';
         }
-        expression += `-${action.options.key}:"${action.options.value}"`;
+        expression += `NOT ${action.options.key}:"${action.options.value}"`;
```

Comment:
I changed this to be consistent with other places in elasticsearch code where we are adding `filter out` filter.

Reply:
Haha, I already started with a question why we are changing this. But makes sense, even though https://www.elastic.co/guide/en/elasticsearch/reference/8.5/query-dsl-query-string-query.html#_boolean_operators states that `-` is a `preferred operator`.

Reply:
Nice find. Let me change this back and change my solution to `-` as well.